### DISCUSSION TITLE
AIR runtime - version 33.1.1.533 ??? Where ist 33.1.1.758

### DISCUSSION THREAD

**Discussion by RGerhardt-Pressmind** [airsdk/Adobe-Runtime-Support#1677] [General]
Hello everyone, yesterday you could download the runtime still in the version 33.1.1.758, I even still have on my computer the .dmg file. Today again only the 33.1.1.533. Why?

**Yesterday (17.02.2022)**
![Today](https://pm-datastore-archiv.s3-eu-west-1.amazonaws.com/logs/Adobe_AIR_SDK_from_HARMAN_old.jpg)

**Today (18.02.2022)**
![Yesterday](https://pm-datastore-archiv.s3-eu-west-1.amazonaws.com/logs/Adobe_AIR_SDK_from_HARMAN_%F0%9F%94%8A.jpg)

**Comment by ajwfrost** [comment]
Looks like some admin going on with the webserver machine as part of a migration they're doing, something's not quite right in the build/version that is on there now. I'm checking and will try to get that .758 version back up asap...

thanks

**Reply by ajwfrost** [reply]
Back working now... thanks for letting me know @RGerhardt-Pressmind

**Reply by GeneralVimes** [reply]
And https://airsdk.harman.com/download shows that .743 is the latest version and suggest to download it, not .758

**Reply by ajwfrost** [reply]
Yes that's weird .. we don't have a 33.1.1.758 version of the shared runtime, there is a version 33.1.1.758 available for the SDK from the release notes page but not from the normal downloads and it's not a version we packaged up for the shared runtime.

So .. the site looks correct to me at the moment, build 743 available by default from both SDK and runtime pages.

thanks

**Reply by P2b10** [reply]
Hi Andrew:

Will 743 will work for what you said on an earlier post?

QUOTE:

download the Android SDK - simplest way of doing this is to install Android Studio and get it to pull down the latest SDK. Then you can set up the path to the SDK within the configuration file (AIR SDK installation folder -> lib/adt.cfg), or if you install it in the default location then the 33.1.1.758 build should find it automatically. Then the build will proceed using Gradle and the same tools as you get with Android Studio.
